packages/core/src/lib/components/T/utils/useProps.ts

```diff
@@ -1,7 +1,7 @@
 import { useThrelte } from '../../../hooks/useThrelte'
 import { resolvePropertyPath } from './resolvePropertyPath'
 
-const ignoredProps = new Set(['$$scope', '$$slots', 'type', 'args', 'attach', 'instance'])
```

Comment:
Should we maybe keeping this in until Svelte 6? It can technically still exist since slots aren't removed, just deprecated